src/flow.h

```diff
@@ -503,6 +505,7 @@ typedef struct Flow_
     uint64_t todstbytecnt;
     uint64_t tosrcbytecnt;
 
+    bool elephant;
```

Comment:
It hurts to increase this structure...

Reply:
Did you check all flags were used ?

Reply:
Seemed like that. Will check again.

Reply:
yes they're all used

Reply:
Do you have suggestions on where to store this then?

Reply:
Why do we need to store it ?
Can we not have a function FlowIsElephant(Flow *f) ?

Reply:
`pahole` shows 2 gaps -- 2 and 4 bytes respectively.

Reply:
> Why do we need to store it ? Can we not have a function FlowIsElephant(Flow *f) ?

I thought about it but then wondered if adding more to calculations each time byte count was incremented would be too much and ended up storing it in a boolean. Do you think it is ok to do that?

Reply:
> `pahole` shows 2 gaps -- 2 and 4 bytes respectively.

Thank you. Will keep in mind to check that if I don't find a better solution.